src/openrct2-ui/windows/Multiplayer.cpp

```diff
@@ -342,12 +342,12 @@ namespace OpenRCT2::Ui::Windows
             case WINDOW_MULTIPLAYER_PAGE_INFORMATION:
             {
                 auto size = _windowInformationSize ? _windowInformationSize.value() : InformationGetSize();
-                WindowSetResize(*this, size.x, size.y, size.x, size.y);
+                WindowSetResize(*this, { size.x, size.y }, { size.x, size.y });
```

Comment:
idem

Reply:
Sadly, these don't use ScreenSize structs... yet. Maybe they should!